src/components/ButtonGroup/ButtonGroup.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import classNames from "classnames";
+
+import "./_button-group.scss";
+
+const ButtonGroup = ({ buttons, label, activeButton, setActiveButton }) => {
+  return (
+    <div className="p-button-group">
+      <div className="p-button-group__inner">
+        <span className="p-button-group__label">{label}</span>
+        <div className="p-button-group__buttons">
+          {buttons.map((label) => (
+            <button
+              aria-label={`view by ${label}`}
```

Comment:
```suggestion
              aria-label={`View by ${label}`}
```

Reply:
Screen readers are not the only assistive technology to use ARIA. However, some screen readers use capitalisation as cues for intonation.

Reply:
Cool, I'll make a note to go update the rest of them in a follow-up.